src/features/game/lib/gameMachine.ts

```diff
@@ -219,6 +224,14 @@ export function startGame(authContext: Options) {
             },
           },
         },
+        announcing: {
+          on: {
+            ACKNOWLEDGE: {
+              actions: [() => acknowledgeRead()],
```

Comment:
Same as above, you can just pass the signature here as well. Also for consistency, can we set `actions` after `target` 😄